dist/api_data.py

```diff
@@ -2019,6 +2019,10 @@ def __ge__(self, other):
     Config('dryrun', 'false', r'''
         perform the checks associated with RTS, but don't modify any data.''',
         type='boolean'),
+    Config('threads', '4', r'''
+        maximum number of threads WiredTiger will start to help rollback to stable. Each
+        rts worker thread uses a session from the configured session_max''',
+        min=0, max=10),
```

Comment:
I think min should be 1, which equals to the configure of 0 currently?

Reply:
At first, I want to leave a option to configure 0 workers. It can be useful to disable the worker logic if there is a bug in it.

Reply:
I don't understand the point of doing that. We default to 4, so most of our testing will get the new behaviour - this is good since we want to test it. But if there's a bug, we're probably going to revert this PR, right?

The only way I can see this being useful is if we're going to backport it to the stable branches straight away. Do you know whether this is the case @kommiharibabu?

Reply:
We are not planning to backport it.

After thinking about it for some time, I agree with you. There is no code flow that is different to let an issue happen only with the workers compared to the original code flow. I will change the minimum value to 1.

Reply:
I think  the value 0 is fine, it should mean the main thread will do the job?

Reply:
Yes, We need the code change of 0 workers, because RTS is called at the end of the salvage operation on a single collection. Using workers, there is just an overhead. I reverted the change to make the minimum number of workers 1.